src/app_service/service/wallet_account/service_token.nim

```diff
@@ -166,6 +169,7 @@ proc checkRecentHistory*(self: Service, addresses: seq[string]) =
     error "error: ", errDescription
 
 proc reloadAccountTokens*(self: Service) =
+  self.events.emit(SIGNAL_WALLET_RESET_RELOAD_TIMER, Args())
```

Comment:
This signal is emitted, but not handled in any module.
As I see you intended to use it to reset `lastReloadTimestamp`, but basically you don't need it at all, cause `lastReloadTimestamp` is the view-only variable, used for the tooltip of reload button, so you can reset it in the view directly when you're calling `reloadAccountTokens` function, but even that is not needed, cause it will be just overwritten with the new value once the reload process is done. So feel free to remove this line as well as this signal.